enterprise/server/raft/filestore/filestore.go

```diff
@@ -164,6 +173,18 @@ func (pmk *PebbleKey) Bytes(version PebbleKeyVersion) ([]byte, error) {
 		partDir := PartitionDirectoryPrefix + pmk.partID
 		filePath = filepath.Join(partDir, filePath, "v2")
 		return []byte(filePath), nil
+	case Version3:
+		rih := pmk.remoteInstanceHash
+		if pmk.isolation == "ac" && rih == "" {
+			rih = "0"
```

Comment:
uber-crazy, feel-free-to-ignore-this nit: is this value functionally impossible as a remote instance hash, or just extremely unlikely?

Reply:
I'm not sure actually, but if it's a possible output then it's as likely/unlikely as any other collision of the hash of (remote instance name).